infrahub_sdk/graphql/utils.py

```diff
@@ -0,0 +1,30 @@
+import ast
+
+
+def get_class_def_index(module: ast.Module) -> int:
+    """Get the index of the first class definition in the module.
+    It's useful to insert other classes before the first class definition."""
+    for idx, item in enumerate(module.body):
+        if isinstance(item, ast.ClassDef):
+            return idx
+    return -1
+
+
+def insert_fragments_inline(module: ast.Module, fragment: ast.Module) -> ast.Module:
+    """Insert the Pydantic classes for the fragments inline into the module."""
+    module_class_def_index = get_class_def_index(module)
+
+    fragment_classes: list[ast.ClassDef] = [item for item in fragment.body if isinstance(item, ast.ClassDef)]
+    for idx, item in enumerate(fragment_classes):
+        module.body.insert(module_class_def_index + idx, item)
```

Comment:
What would this look like if `get_class_def_index()` returns a -1. The first entry gets added to the end and then the other fragments gets added starting from the beginning?